php/README.md

```diff
@@ -4,6 +4,16 @@ This documentation describes the **PHP** coding standards for all Seegno project
 
 Our main focus will be the use of [Symfony2](http://symfony.com/) framework.
 
+## Base Standards
+
+We follow the standards described in these documents:
+- [Symfony Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html)
```

Comment:
I think this should be in a separate subtopic since the Symfony documentation already states that "Symfony follows the standards defined in the PSR-0, PSR-1 and PSR-2 documents.".

That page simply showcases examples applied to Symfony code and therefore does not constitute itself a standard.